### ISSUE TITLE
Python version mismatch

### ISSUE THREAD

**Issue by euvl** [ProfHoekstra/bluesky#96]
Hi, 

You are doing a great job guys, really nice tool! 

Have a small issue here, I've tried to run the bluesky on python **v2.7.13** and the `check.py` fails to start with this error:

```
Traceback (most recent call last):
  File "check.py", line 133, in <module>
    from bluesky import *
  File "/Users/yev/Projects/sparkl/bluesky/bluesky/__init__.py", line 8, in <module>
    import bluesky.ui.pygame.screen as _scr
  File "/Users/yev/Projects/sparkl/bluesky/bluesky/ui/pygame/__init__.py", line 1, in <module>
    from .screen import Screen
  File "/Users/yev/Projects/sparkl/bluesky/bluesky/ui/pygame/screen.py", line 21, in <module>
    from .dialog import fileopen as opendialog
  File "/Users/yev/Projects/sparkl/bluesky/bluesky/ui/pygame/dialog.py", line 7, in <module>
    from tkinter import *
ImportError: No module named tkinter
```

From the python documentation, I've found this:
```
Install Tkinter

To see if you have Tkinter, type python on the Linux command line. 
Then at the Python prompt, type

>>> import Tkinter

Or in Python 3

>>> import tkinter
````

Seems that the code refers to **python3** tkinter library while documentation says "BlueSky is developed in Python 2.x".

**Comment by ProfHoekstra** [comment]
Dear Yev,

Thanks for reporting this. You are right, we have made BlueSky Python 3.6 compatible. But it should also work on 2.7. When I run Python 2.7 I can import tkinter also in lower case, I also have 2.7.13 …..

But on internet I can see more people have this problem. So we’ll investigate.

Best regards,
    Jacco Hoekstra

**Comment by jooste** [comment]
Fixed in d8b1818.
